src/Config.ts

```diff
@@ -22,6 +22,44 @@ export class SocketOptions {
     //TO-DO
 }
 
+export interface SslOptions {
```

Comment:
There is a similar interface in `tls` module called `ConnectionOptions`, can you look at the source and re-evaluate, I don't know why they did it interface?